components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/gen/java/org/wso2/carbon/identity/api/server/application/management/v1/PassiveStsConfiguration.java

```diff
@@ -75,6 +76,24 @@ public void setReplyTo(String replyTo) {
         this.replyTo = replyTo;
     }
 
+    /**
+    **/
+    public PassiveStsConfiguration replyToLogout(String replyToLogout) {
+
+        this.replyToLogout = replyToLogout;
+        return this;
+    }
+    
+    @ApiModelProperty(value = "")
+    @JsonProperty("replyToLogout")
+    @Valid
+    public String getReplyToLogout() {
+        return replyToLogout;
+    }
+    public void setReplyToLogout(String replyToLogout) {
+        this.replyToLogout = replyToLogout;
+    }
+
```

Comment:
Remove unwanted new line

Reply:
This is an auto-generated file from applications.yaml[1]. Therefore, no manual changes were done.

[1] https://github.com/wso2/identity-api-server/blob/master/components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml